StPetersburg/IVEf55.xml

```diff
@@ -0,0 +1,265 @@
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0301IV"/>
+                        <idno>IV Ef. 55</idno>
+                        <altIdentifier>
+                            <idno>Platonov cat. II.19</idno>
+                        </altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                          <locus from="3r" to="72v"></locus>
+                          <title type="complete"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <incipit xml:lang="gez">እዝል፡ መስተጋብዕ፡ ዘሠኑይ፡ ወልቡ፡ </incipit>
+                            <note>With musical notation</note>
+
+                              <msItem xml:id="ms_i1.1">
+                                <locus from="3v" to="4r"></locus>
+                                <title type="complete" ref="LIT4518Hymn"></title>
+                                <textLang mainLang="gez"></textLang>
+                                <note>With musical notation</note>
+                              </msItem>
+                              <msItem xml:id="ms_i1.2">
+                                <locus from="3r" to="7r"></locus>
+                                <title type="incomplete" ref="LIT2000Mazmur"></title>
+                                <textLang mainLang="gez"></textLang>
+                                <note>With musical notation</note>
+                              </msItem>
+                              <msItem xml:id="ms_i1.3">
+                                <locus from="7r" to="7v"></locus>
+                                <title type="incomplete" ref="LIT2637Deuteronomy"></title>
+                                <textLang mainLang="gez"></textLang>
+                                <note>With musical notation</note>
+                              </msItem>
+                              <msItem xml:id="ms_i1.4">
+                                <locus from="8r" to="10r"></locus>
+                                <title type="incomplete" ref="LIT2000Mazmur"></title>
+                                <textLang mainLang="gez"></textLang>
+                                <note>With musical notation</note>
+                              </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
```

Comment:
Disambiguating titles should be inserted, or the printed version would lead to an impression that there are several Psalters inside, not several single psalms or sets of psalms